Source/WebCore/dom/Node.h

```diff
@@ -328,11 +328,11 @@ class Node : public EventTarget, public CanMakeCheckedPtr<Node> {
     bool selfOrPrecedingNodesAffectDirAuto() const { return hasStateFlag(StateFlag::SelfOrPrecedingNodesAffectDirAuto); }
     void setSelfOrPrecedingNodesAffectDirAuto(bool flag) { setStateFlag(StateFlag::SelfOrPrecedingNodesAffectDirAuto, flag); }
 
-    TextDirection effectiveTextDirection() const;
-    void setEffectiveTextDirection(TextDirection);
+    TextDirection textDirection() const;
```

Comment:
I don't think this rename is an improvement.

Reply:
This change was reverted.